src/main/java/animals/fish/Fish.java

```diff
@@ -1,30 +1,13 @@
 package animals.fish;
 
-import animals.Animals;
+import animals.Animal;
 
-public abstract class Fish extends Animals {
+public abstract class Fish extends Animal {
 
     public Fish (String animalName, String typeOfFood) {
         super(animalName, typeOfFood);
     }
 
-    @Override
```

Comment:
This is what I was talking about in my initial comment: this is where those methods belong, you shouldn't have deleted them from here